presto-hive/src/main/java/io/prestosql/plugin/hive/HdfsEnvironment.java

```diff
@@ -37,6 +39,7 @@
 {
     static {
         HadoopNative.requireHadoopNative();
+        FileSystemManager.registerCache(PrestoFileSystemCache.INSTANCE);
```

Comment:
sometime back we had added `requireHadoopNative` call at one more place where hadoop code is initialized: https://github.com/prestosql/presto/pull/5076 We should register the cache there as well?

Reply:
This is a good point, but it shouldn't be needed there, since the Kerberos code doesn't touch any file system related code.